### PR TITLE
Refactor/taxonomy/removeredux

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: webapp/components/survey/TaxonomyDetails/store/actions/useUpdate.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { useDispatch } from 'react-redux'
 import { debounceAction } from '@webapp/utils/reduxUtils'
 import * as Taxonomy from '@core/survey/taxonomy'
 
-import { useSurveyId, TaxonomiesActions } from '@webapp/store/survey'
+import { useSurveyId, TaxonomiesActions, SurveyActions } from '@webapp/store/survey'
```
Comment (minotogna) [inline webapp/components/survey/TaxonomyDetails/store/actions/useUpdate.js:8]: do you still need TaxonomiesActions ?

File: webapp/components/survey/TaxonomyDetails/store/state/state.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ import * as R from 'ramda'
 import * as Validation from '@core/validation/validation'
 import * as ObjectUtils from '@core/objectUtils'
 
+export const Taxonomykeys = {
```
Comment (minotogna) [inline webapp/components/survey/TaxonomyDetails/store/state/state.js:7]: not clear what those keys are. can u explain ?

File: webapp/components/survey/TaxonomyList/HeaderLeft/HeaderLeft.js [hunk 3]
```diff
@@ -13,13 +13,13 @@ const HeaderLeft = (props) => {
   const inTaxonomiesPath = Boolean(matchPath(pathname, appModuleUri(designerModules.taxonomies)))
 
   const { headerProps } = props
-  const { onTaxonomyCreated } = headerProps
+  const { onNewTaxonomy } = headerProps
```
Comment (minotogna) [inline webapp/components/survey/TaxonomyList/HeaderLeft/HeaderLeft.js:16]: **onTaxonomyCreate** might be better  more consistent

File: webapp/components/survey/TaxonomyDetails/store/state/state.js (hunk 2 above)
Comment (ramirobg94) [reply webapp/components/survey/TaxonomyDetails/store/state/state.js:7]: not sure about the name:

```
<Table
            module={State.Taxonomykeys.taxa}
```

Maybe this keys should be in other parts of the code.

Comment (minotogna) [reply webapp/components/survey/TaxonomyDetails/store/state/state.js:7]: is it used only there? then no need for extra keys object

Comment (ramirobg94) [reply webapp/components/survey/TaxonomyDetails/store/state/state.js:7]: just spotted that i can use a string as we did in other parts! thanks!

**Pull request merged**